app/assets/stylesheets/more-articles.scss

```diff
@@ -149,8 +149,10 @@
       margin-bottom: 30px;
     }
     .cta{
-      padding: 7px 9px;
-      margin-top: 10px;
+      width: 100px;
+      /*padding: 7px 9px;*/
```

Comment:
Let's remove commented-out CSS :)

Reply:
Oops! Will take care of it right away lol